### PR TITLE
DBZ-2645 Add Vitess connector documentation

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-2645

Hi @gunnarmorling, is there a way to build the documentation and website locally to check if everything works? Thanks.

### CODE REVIEW

**Comment by keweishang** [comment]
I used https://github.com/debezium/debezium.github.io to build the website locally that contains the documentation from the [main debezium repo](https://github.com/debezium/debezium). It looks good to me.

File: documentation/modules/ROOT/pages/connectors/vitess.adoc [hunk 1]
```diff
@@ -0,0 +1,1447 @@
+// Category: debezium-using
+// Type: assembly
+[id="debezium-connector-for-vitess"]
+= {prodname} connector for Vitess
+
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/vitess.adoc:5]:
```suggestion
:context: vitess
```

**Comment by Naros** [comment]
We had some recent changes to the documentation as a part of DBZ-2497.  I've requested one tiny change above that should address the `{context}` variable used in links as part of the shared metrics templates.  If I may ask, which connector did you use as a basis for the vitess.adoc?  I just want to make sure that there aren't any other changes needed.

**Comment by keweishang** [comment]
@Naros Thanks for reviewing. I've just committed the suggested change. I've used `postgresql.adoc` as the basis for the `vitess.adoc`. Let me know if other changes are needed.

**Review by gunnarmorling — COMMENTED** [review]
Wow, great work! Thanks a lot, @keweishang! One comment inline.

File: documentation/modules/ROOT/pages/connectors/vitess.adoc [hunk 2]
```diff
@@ -0,0 +1,1447 @@
...
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname}'s Vitess connector captures row-level changes in the shards of a Vitess link:https://vitess.io/docs/concepts/keyspace/[keyspace]. Vitess versions 7 and 8 are supported.
+endif::community[]
+ifdef::product[]
```
[the remark this answers is not in the document]
Comment (keweishang) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:18]: Thank you for pointing this out. I actually had the same question. I'll remove the product `ifdef` parts.

[the remark this answers is not in the document]
Comment (keweishang) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc]: I removed all the `ifdef::product[]`s. I also kept the content of the `ifdef::community[]`s, but removed the `ifdef::community[]` logic.

**Comment by jpechane** [comment]
@keweishang Great job! Applied

**Comment by keweishang** [comment]
Awesome! Thank you @jpechane

**Pull request merged**